dist/index.html

```diff
@@ -9,6 +9,22 @@
 
   <body>
     <header>
+<<<<<<< HEAD
+      <h2>SHOWNET</h2>
+      <nav>
+        <ul class="page-navigation">
+          <li><a href="#" class="nav-link">Home</a></li>
+          <li><a href="#" class="nav-link">Popular</a></li>
+          <li><a href="#" class="nav-link">Genre</a></li>
+        </ul>
+      </nav>
+      <div class="icon-input">
+        <input type="text" placeholder="Search" name="movie search" class="search" />
+        <i class="fa-solid fa-magnifying-glass"></i>
+      </div>
+    </header>
+    <div class="cover"></div>
+=======
```

Comment:
`<<<<<<HEAD` and `=======` are not part of the HTML code, and this may tamper with the efficiency and neatness of your code, please remove it